conf/test/tools.config

```diff
@@ -16,6 +16,7 @@ params {
     germline_resource      = params.test_data['homo_sapiens']['genome']['gnomad_r2_1_1_21_vcf_gz']
     intervals              = params.test_data['homo_sapiens']['genome']['genome_21_multi_interval_bed']
     pon                    = params.test_data['homo_sapiens']['genome']['mills_and_1000g_indels_21_vcf_gz']
+    ngscheckmate_bed       = "https://raw.githubusercontent.com/parklab/NGSCheckMate/master/SNP/SNP_GRCh38_hg38_wChr.bed"
```

Comment:
shouldn't we need a trimmed version of this file with only the intervals relevant to our test data?

Reply:
The tool essentially ignores positions that are empty. 
One thing that we sometimes do on targeted mutation panels is to intersect the bed files, that is something I have considered implementing here.

Reply:
I was more thinking about having it on a smaller size so we can put it in the test-dataset repo, so that it can be downloaded and we could even test offline

Reply:
Yeah, it would be good to add to the test_data, I did think that. The whole bed file is <1Mb, but chr21 would obviously be even smaller

Reply:
And it doesn't need to be in both locations, only the one the test is actually using...

Reply:
And also can you add it test.config instead?